### PR TITLE
[ENV] Added comments and removed duplicate file

### PR DESCRIPTION
**Pull request by lumos309**

### CODE REVIEW

**Comment by lumos309** [comment]
No functional changes - only cleanup and comments.

File: public/externalLibs/env_visualizer/visualizer.js [hunk 1]
```diff
@@ -145,11 +149,14 @@
     viewport.render();
   }
   
+
+	/* Not used right now (no use for clicking data objects).
```
Comment (jiachen247) [inline public/externalLibs/env_visualizer/visualizer.js]: can we remove this if we are not using it!

File: public/externalLibs/env_visualizer/visualizer.js [hunk 2]
```diff
@@ -158,16 +165,15 @@
     viewport.render();
   }
 
-  function drawHitFrames() {
+  /* Not used now (no use for clicking frames).
```
Comment (jiachen247) [inline public/externalLibs/env_visualizer/visualizer.js]: same!

File: public/externalLibs/env_visualizer/visualizer.js [hunk 3]
```diff
@@ -123,6 +123,10 @@
    */
   var builtinsToDraw = [];
 
+	/**
```
Comment (jiachen247) [inline public/externalLibs/env_visualizer/visualizer.js]: can we format this nicely please!

File: public/externalLibs/env_visualizer/visualizer.js [hunk 4]
```diff
@@ -197,6 +203,12 @@
     viewport.render();
   }
 
+	/**
+	 * The actual drawing functions.
+	 * "Scene" objects are the actual visible drawings.
+	 * "Hit" objects are the hitboxes (for mouseovers/clicks) for each scene.
+	 */
+
```
Comment (jiachen247) [inline public/externalLibs/env_visualizer/visualizer.js]: fix indentation and remove this line.

**Comment by openorclose** [comment]
https://github.com/source-academy/cadet-frontend/tree/master/public/externalLibs/env_visualizer

https://github.com/source-academy/cadet-frontend/blob/aa7d4806be91e1c55ada34cc092b401c118bcf30/public/externalLibs/index.js#L44

These file names should be the same also.

**Comment by jiachen247** [comment]
can remove this file as well!! `/src/components/workspace/ControlBar.tsx.orig`

**Pull request merged**